graphql_compiler/macros/macro_expansion.py

```diff
@@ -125,13 +125,21 @@ def _merge_selection_into_target(target_ast, selection_ast, subclass_sets=None):
 
     Args:
         target_ast: AST at the @macro_edge_target directive
+        type_at_target: str name of the type at the target
```

Comment:
I believe everywhere else we just pass around GraphQL type objects around, why not pass a type object here as well?